apps/ui/src/components/checkbox/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface CheckboxProperties {
```

Comment:
I recommend using `type` instead of `interface` as follow
```ts
export type Properties = {
  id?: string;
  label?: string;
  checked?: boolean;
  disabled?: boolean;
  onChange: (checked: boolean) => void;
}
```